keyboards/viktus/minne/info.json

```diff
@@ -0,0 +1,57 @@
+{
+    "manufacturer": "Viktus Design LLC",
+    "keyboard_name": "Viktus Minne",
+    "maintainer": "BlindAssassin111",
+    "url": "https://viktus.design",
+    "usb": {
+        "device_version": "1.0.0",
+        "vid": "0x5644",
+        "pid": "0x4D49"
+    },
+    "bootloader": "atmel-dfu",
+    "processor": "atmega32u4",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": false,
+        "mousekey": false,
+        "nkro": true
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["B7", "F7", "D5", "D3", "D2", "D1", "D0"],
+        "rows": ["B6", "B5", "B4", "D7", "D6", "D4", "C6", "C7"]
+    },
+    "rgblight": {
+        "led_count": 14,
+        "pin": "F6",
+        "max_brightness": 191,
+        "animations": {
+            "all": true
```

Comment:
No longer a valid option, animations must be explicitly listed

Reply:
Ahh, I followed the docs and it was listed so didn't know.

Reply:
Got a link? we might have to go update anything that slipped through.

Reply:
Listed in both of these locations:

https://docs.qmk.fm/#/reference_info_json?id=rgblight-animations
https://docs.qmk.fm/#/reference_info_json?id=infojson-format

Reply:
ANnnnd its gone, cheers!